### PR TITLE
RELATED: ONE-5027 Update documentation for popup legend

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/723/ : SUCCESS in 1m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/459/ : SUCCESS in 7s

File: docs/15_props__chart_config.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ Within one visualization, the `colorMapping` property overrides the `colorPalett
 
     ![PopUp Legend](assets/top_legend.png "PopUp Legend Component")
 
-    **NOTE:** When `config.legend.responsive` is set to `"autoPositionWithPopup"`, the `config.legend.position` property is ignored.
+    **NOTE:** When `config.legend.responsive` is set to `"autoPositionWithPopup"`, the `config.legend.position` property is ignored for some container sizes and the position is automatically adjusted.
```
Comment (evgeniabogranskaya) [inline docs/15_props__chart_config.md:242]: **NOTE:** When `config.legend.responsive` is set to `"autoPositionWithPopup"`, the `config.legend.position` property may be ignored for containers of a smaller size, and the legend position will be automatically adjusted anyway.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/726/ : SUCCESS in 1m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/461/ : SUCCESS in 9s

**Comment by evgeniabogranskaya** [comment]
LGTM

**Review by ivan-nejezchleb — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR416

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/727/ : SUCCESS in 1m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/115/ : SUCCESS in 43s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/17/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/206/ : SUCCESS in 20s

**Pull request merged**